src/clock.rs

```diff
@@ -8,31 +9,31 @@ use crate::time::Hertz;
 /// longer be changed.
 #[derive(Clone, Copy)]
 pub struct Clocks {
-    cpu: Hertz,
-    apb0: Hertz,
+    pub(crate) aclk: Hertz,
+    pub(crate) apb0: Hertz,
 }
 
 impl Clocks {
     #[doc(hidden)]
     pub fn new() -> Self {
-/*
-        [MAIXPY]Pll0:freq:806000000
-        [MAIXPY]Pll1:freq:398666666
-        [MAIXPY]Pll2:freq:45066666
-        [MAIXPY]cpu:freq:403000000
-        [MAIXPY]kpu:freq:398666666
-        in freq: 26000000
-        cpu_freq: 390000000
-*/
+        /*
+                [MAIXPY]Pll0:freq:806000000
+                [MAIXPY]Pll1:freq:398666666
+                [MAIXPY]Pll2:freq:45066666
+                [MAIXPY]cpu:freq:403000000
+                [MAIXPY]kpu:freq:398666666
+                in freq: 26000000
+                cpu_freq: 390000000
+        */
         Self {
-            cpu: Hertz(403_000_000),
```

Comment:
I think after k210 startup, the output frequency of pll0 is 780MHz, and for aclk(cpu) it is 390MHz, so the origin data here might be wrong.
However, for `UARTHS`, both `403_000_000` and `390_000_000`  works well here.